src/app/components/info-hub/pages/guidelines/guidelines.js

```diff
@@ -0,0 +1,128 @@
+import styles from "../welcome/welcome.css"
+import Bubble from "../../../bubble/bubble"
```

Comment:
Overall great job on the layout of this page. It looks pretty much done, but the text is hiding behind the NavBar when I see this page. Nishant had this problem when he was making this change so the two of you should make the CSS changes to make this not happen for other pages in the future.